app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
Adding instances to the `people` dictionary in the `__init__` method is not recommended. The creation of an object and its registration should be separate processes. It would be better to handle this in a separate method or outside of the class.